src/components/Info.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Info = ({title, date, location, logo}) => (
+  <>
+    <div className="logo">
+      <img alt={logo.alt} src={require(`../images/${logo.path}`)}/>
+    </div>
+
+    <div className="text">
+      <h1 className="title">{title}</h1>
+      <h2 className="date">{date}</h2>
+      <h3 className="location">{location}</h3>
+    </div>
+  </>
+);
+
+Info.propTypes = {
+  title: PropTypes.string.isRequired,
+  date: PropTypes.string,
+  location: PropTypes.string,
```

Comment:
Both date and location should be required or provided with a default value.

Reply:
If I don't want to specify the date, why not just null?

Reply:
Add null as a default value to be explicit then.

Reply:
agree, it is missing the `defaultProps`

Reply:
I don't agree we should do that. I think it is obvious that if you don't pass any value, null is going to be used.

Expose your point of view, please.

Reply:
`null` violates any type "system" JS might have. It fits every type. Since we are using it in here, I think we should make it explicit.

We are relying on something that is the cause of many bugs and, quite honestly, shouldn't exist. We should be explicit about it.

Reply:
But I am requiring it to be a string or to not be anything.

Reply:
Yet it begs the question: What is the default if no string is provided? Null? Empty string?
Explicit is better than implicit in almost every case and I think it applies here. 
Besides, by adding that everyone reading the code knows you are expecting a default value (even if it's null) and you didn't forget to add those fields to the `defaultProps`.

Reply:
By making it optional, I leave it to the user of the component the possibility to have or not date and location to be displayed. If you don't provide a value for those, then it is `null` and nothing will be rendered. It doesn't raise the question for me.

I don't think add `null` as a default prop is a common practice but I can do it anyway.

Reply:
I don't know if JS's default behaviour is using null or an empty string. That question would arise for me. Ii it's explicit, it's obvious what the default value is.